ext/ddtrace.stub.php

```diff
@@ -23,6 +23,40 @@
      */
     const DBM_PROPAGATION_FULL = UNKNOWN;
 
+    class SpanLink implements \JsonSerializable {
+        /**
+         * @var string $traceId A 32-character, lower-case hexadecimal encoded string of the linked trace ID. This field
+         * shouldn't be directly assigned. Use the SpanData::getLinks() method instead.
```

Comment:
```suggestion
         * shouldn't be directly assigned an id from SpanData. Use the SpanData::getLinks() method instead.
```
directly assigning it when getting it from distributed tracing or such is fine, just not assigning a decimal id.